components/butterfly/protocols/swim.proto

```diff
@@ -51,6 +51,7 @@ message Service {
   optional string pkg = 9;
   optional bytes cfg = 10;
   optional SysInfo sys = 12;
+  optional string channel = 13;
```

Comment:
If we leave it out of the gossip, it won't work. The channel will always be empty.